tests/Session/test_profile.py

```diff
@@ -48,5 +56,13 @@ def testListFoodStorages(self):
         self.profile.addFoodStorage(FRIDGE, fridge_name)
         self.assertEqual([freezer_name, fridge_name], self.profile.listFoodStorages(FOOD_STORAGES)) 
 
+    def testAddFoods(self):
+        # Check self.profile.addFoods returns 0 = tests that food storage is not open yet 
```

Comment:
I think you forgot to do this check

Reply:
Yeahhh not sure what happened here tbh lol. I fixed it.